syntax = "proto3";

package hello;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// Message Definition
message HelloRequest {
  string name = 1;
  int32 num = 2;
  bool has_boolean = 3;
}

message HelloReply {
  string message = 1;
  int32 id = 2;
  bool has_boolean = 3;
}